js/content.js

```diff
@@ -305,14 +312,61 @@ function stopCSSInterval() {
     clearInterval(intervalCSSId);
 }
 
-//Create mutation observer
+function spinnerAnimation() {
```

Comment:
I don't think the method name describes what the method does well. Could it be named to something like `addSpinnerAnimation`?